content/en/news/releases/1.8.x/announcing-1.8/change-notes/index.md

```diff
@@ -0,0 +1,186 @@
+---
+title: Change Notes
+description: Istio 1.8 release notes.
+weight: 10
+---
+
+## Traffic Management
+
+- **Added** DNS capture in istio-agent by default for VMs installed using `istioctl x workload entry configure`.
+
+- **Added** support for injecting `istio-cni` into `k8s.v1.cni.cncf.io/networks` annotation with preexisting value that uses JSON notation.
+  ([Issue #25744](https://github.com/istio/istio/issues/25744))
```

Comment:
AFAIK this doesn't work and shouldn't be added to release notes. @howardjohn FYI.

Reply:
cc @luksa as the original PR author - commenting out this release note until confirmed